elements/noflo-ui.html

```diff
@@ -480,10 +477,30 @@
           this.hideAlert();
         }.bind(this), 1300);
       },
+      toggleEditAsProject: function (enable) {
```

Comment:
`showEditAsProject` would be better name. Toggle implies that it switches/toggles state when called